src/index.ts

```diff
@@ -2469,6 +2469,57 @@ const configShardusNetworkTransactions = (): void => {
       return false
     }
   )
+  shardus.registerShutdownHandler(
+    'nodeInitReward',
+    (node: P2P.NodeListTypes.Node, record: P2P.CycleCreatorTypes.CycleRecord) => {
+      if (record.activated.includes(node.id)) {
+        if (record.txadd.some((entry) => entry.txData.nodeId === node.id && entry.type === 'nodeInitReward')) {
+          console.warn(`shutdown condition: active node with id ${node.id} is already in txadd (nodeInitReward); this should not happen`)
```

Comment:
should just use console.log instead of warn.  Should be okay to put this behind ShardeumFlags.VerboseLogs